Package.swift

```diff
@@ -6,7 +6,7 @@ let package = Package(
   platforms: [.macOS(.v10_13)],
   products: [
     .executable(name: "AirbnbSwiftFormatTool", targets: ["AirbnbSwiftFormatTool"]),
-    .plugin(name: "AirbnbSwiftFormatPlugin", targets: ["AirbnbSwiftFormatPlugin"]),
+    .plugin(name: "FormatSwift", targets: ["FormatSwift"]),
```

Comment:
Xcode makes clear that this plugin is from our package, so we can have a simpler name.
<img width="343" alt="Screen Shot 2022-07-21 at 12 13 06 PM" src="https://user-images.githubusercontent.com/1791049/180297332-5c35d3b7-6d80-4d8e-a546-1dede158997a.png">